File: app/src/main/java/com/kota/Bahamut/Pages/InstructionsPage.java
package com.kota.Bahamut.Pages;

import com.kota.Bahamut.BahamutPage;
import com.kota.Bahamut.R;
import com.kota.TelnetUI.TelnetPage;

public class InstructionsPage extends TelnetPage {
    @Override
    public int getPageType() {
        return BahamutPage.BAHAMUT_INSTRUCTIONS;
    }

    @Override
    public int getPageLayout() { return R.layout.instruction_page; }

    @Override // com.kota.TelnetUI.TelnetPage
    public boolean isPopupPage() {
        return true;
    }

    @Override // com.kota.TelnetUI.TelnetPage
    public boolean isKeepOnOffline() {
        return true;
    }

    @Override
    public void onPageDidLoad() {

    }
}

File: app/src/main/java/com/kota/Bahamut/Pages/InstructionsPage.kt
package com.kota.Bahamut.Pages

import android.annotation.SuppressLint
import android.webkit.WebView
import com.kota.Bahamut.BahamutPage
import com.kota.Bahamut.R
import com.kota.TelnetUI.TelnetPage

class InstructionsPage : TelnetPage() {
    override fun getPageType(): Int {
        return BahamutPage.BAHAMUT_INSTRUCTIONS
    }

    override fun getPageLayout(): Int {
        return R.layout.instruction_page
    }

    // com.kota.TelnetUI.TelnetPage
    override fun isPopupPage(): Boolean {
        return true
    }

    // com.kota.TelnetUI.TelnetPage
    override fun isKeepOnOffline(): Boolean {
        return true
    }

    @SuppressLint("SetJavaScriptEnabled")
    override fun onPageDidLoad() {
        val webView = findViewById(R.id.instruction_web_view) as WebView
//        webView.settings.javaScriptEnabled = true
        webView.loadUrl("https://github.com/kodakjerec/bahamut-BBS-2/blob/main/book/outline.md")
    }
}
